shell/platform/darwin/ios/framework/Source/FlutterKeyboardManager.h

```diff
@@ -0,0 +1,85 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterKeyPrimaryResponder.h"
+
+#import <Foundation/NSObject.h>
+#import <UIKit/UIKit.h>
+
+#include "flutter/fml/memory/weak_ptr.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterKeyPrimaryResponder.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterKeySecondaryResponder.h"
+
+typedef void (^KeyEventCompleteCallback)(bool, UIPress* _Nonnull, UIPressesEvent* _Nonnull)
+    API_AVAILABLE(ios(9.0));
+
+/**
+ * A hub that manages how key events are dispatched to various Flutter key
+ * responders, and whether the event is propagated superclass to propagate it to
```

Comment:
What does "whether the event is propagated superclass to propagate it to other views" mean?

Reply:
Well, there's a missing "to the", as in "to the superclass", but I suspect it's not that clear.  Reworded it.